simple_qa_rnn/relation_prediction/simple_qa_relation.py

```diff
@@ -0,0 +1,72 @@
+import os
+
+from torchtext import data
```

Comment:
Please describe in README how to install torchtext. I think it's currently pre-release software and maybe isn't obvious.